File: Stage6/C/Problem3003.c
#include <stdio.h>

int main() {
    int defaultPieces[6] = {1, 1, 2, 2, 2, 8};
    int piece;

    for (int i = 0; i < 6; i++) {
        scanf("%d", &piece);
        printf("%d ", defaultPieces[i] - piece);
    }

    return 0;
}

File: Stage6/Java/Problem3003.java
package Stage6.Java;

import java.util.Scanner;

public class Problem3003 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] defaultPieces = {1, 1, 2, 2, 2, 8};

        for (int piece : defaultPieces) {
            System.out.print((piece - sc.nextInt()) + " ");
        }

        sc.close();
    }
}
